frontend-employees/src/app/components/time-tracking/time-tracking.component.html

```diff
@@ -1 +1,6 @@
-<p>time-tracking works!</p>
+<div class="time-tracking-container">
+    <button mat-raised-button class="start-button mat-primary" [disabled]="isTimerRunning" (click)="start()">Start</button>
+    <button mat-raised-button class="stop-button mat-warn" (click)="stop()">Stop</button>
+    <div>Arbeitsdauer:</div>
+    <div class="elapsed-time">{{ elapsedTime | secondsToTime }}</div>

```

Comment:
Angular hat schon eine Pipe die das macht https://angular.io/api/common/DatePipe. Du musst davor einfach die Sekunden in Millisekunden umrechnen.
```suggestion
    <div class="elapsed-time">{{ time | date:'HH:mm:ss':'UTC' }}</div>
```